exchanges/bybit/bybit_websocket.go

```diff
@@ -345,19 +345,26 @@ func (by *Bybit) wsProcessWalletPushData(assetType asset.Item, resp []byte) erro
 	if err != nil {
 		return err
 	}
-	accounts := []account.Change{}
+	creds, err := by.GetCredentials(context.Background())
```

Comment:
context.Background() -> context.TODO() or have context as a param with context.TODO() in that call.